components/browse/BrowseContentCard.vue

```diff
@@ -69,6 +74,13 @@
           }
         }
         return texts;
+      },
+      isEntity() {
+        if (this.cardType && this.cardType.sys.id === 'automatedEntityCard') {
```

Comment:
Just return the value of the conditional and remove all following lines from this function.